src/mpc/model.jl

```diff
@@ -92,7 +92,21 @@ function build_mpc!(m::JuMP.AbstractModel, p::MPCInputs)
 	end
 
 	for b in p.s.storage.types.all
-		if p.s.storage.attr[b].size_kw == 0 || p.s.storage.attr[b].size_kwh == 0
+		if b in p.s.storage.types.hydrogen
+			if p.s.storage.attr[b].max_kg == 0
```

Comment:
why is there no constraint here fixing dvStoredEnergy to zero? (compare to src/core/reopt.jl line 210)